src/swarm-topography/Swarm/Game/Scenario/Topography/Structure/Overlay.hs

```diff
@@ -68,14 +68,14 @@ computeMergedArea (OverlayPair pg1 pg2) =
 zipGridRows ::
   Alternative f =>
   AreaDimensions ->
-  OverlayPair (Grid (f a)) ->
+  OverlayPair [[f a]] ->
```

Comment:
What is the rationale for changing `Grid (f a)` to `[[f a]]` here?  It seems like a step backward in terms of generality.

Reply:
Yeah, I had attempted `Grid (f a)` at first, but encountered a bug in `padSouthwest` that was causing structures to get blanked out.  I didn't have time to fully investigate it.  Created #2004 to follow up.